appserver/tests/payara-samples/test-utils/src/main/java/fish/payara/samples/PayaraTestShrinkWrap.java

```diff
@@ -68,8 +69,9 @@ public static JavaArchive getJavaArchive() {
     private static <T extends Archive<T> & ClassContainer<T>> T getArchive(Class<T> archiveType) {
         return ShrinkWrap.create(archiveType)
                 .addClasses(PayaraArquillianTestRunner.class, PayaraTestRunnerDelegate.class, PayaraVersion.class)
-                .addClasses(SincePayara.class, NotMicroCompatible.class)
```

Comment:
CliCommands executed from the server??? Weird.

Reply:
Correct, to support truly remote domain, running on a different machine.